Lib/idlelib/idle_test/test_outwin.py

```diff
@@ -0,0 +1,144 @@
+""" Test idlelib.outwin.
+"""
+
+import unittest
+import unittest.mock as mock
+from tkinter import Tk, Text
+from idlelib.idle_test.mock_tk import Mbox_func
+
+from idlelib import filelist
+from idlelib import outwin
+from test.support import requires
+requires('gui')
+
+
+class OutputWindowTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        requires('gui')
+        cls.root = Tk()
+        cls.root.withdraw()
+        cls.window = outwin.OutputWindow(None, None, None, cls.root)
+        cls.text = Text(cls.root)
+        cls.window.text = cls.text
+
+    @classmethod
+    def tearDownClass(cls):
+        del cls.text, cls.window
+        cls.root.destroy()
+        del cls.root
+
+    def tearDown(self):
+        self.text.delete('1.0', 'end')
+
+    def test_window_title(self):
+        self.assertEqual(self.window.top.title(), "Output")
+
+    def test_ispythonsource(self):
+        self.assertFalse(self.window.ispythonsource("test.py"))
```

Comment:
Maybe add another filename to test, and the docstring that explicit says this will only return `False`